write_guide.rst

```diff
@@ -5,9 +5,11 @@ The OMERO guide is hosted on `readthedocs <https://readthedocs.org/>`_.
 This section describes on to create a new guide.
 
 To create a new guide:
-  - create a new repository on GitHub using the `template guide <https://github.com/ome/guide-template>`_
-  - enable travis to it is automatically built when commits are pushed
-  - add the new repository as a new project on `readthedocs <https://readthedocs.org/>`_
-  - add the new repository as a subproject of `omero-guides <https://readthedocs.org/projects/omero-guides/>`_ using an alias e.g. omero-guide-figure has been added as figure
-  - add it to the this repository as a submodule using the same alias.
-  - insert the guide into one of the existing files or in a new file it is a new section.
+  - Create a new repository on GitHub using the `template guide <https://github.com/ome/guide-template>`_.
+  - Add the new repository as a new project on `readthedocs <https://readthedocs.org/>`_.
+  - Add the new repository as a subproject of `omero-guides <https://readthedocs.org/projects/omero-guides/>`_ using an alias e.g. omero-guide-figure has been added as figure,
+  - Add it to the this repository as a submodule using the same alias.
+  - Insert the guide into one of the existing files or in a new file it is a new section.
+  - If the new repository will contain a ``binder`` folder to generate an environment for analysis,
+  add to the `binder_badge.yml <https://github.com/ome/.github/blob/master/workflow-templates/binder_badge.yml>`_ and `repo2docker.yml <https://github.com/ome/.github/blob/master/workflow-templates/repo2docker.yml>`_ workflows to ``.github/workflows``.
```

Comment:
```suggestion

```

Removing this sentence completely, as I suggest to add it to the previous bullet point to fix formatting, see https://github.com/ome/omero-guides/pull/116/files#r636809563